azure/services/managedclusters/managedclusters.go

```diff
@@ -104,6 +104,7 @@ func (s *Service) Reconcile(ctx context.Context) error {
 		Tags:     *to.StringMapPtr(managedClusterSpec.Tags),
 		ManagedClusterProperties: &containerservice.ManagedClusterProperties{
 			NodeResourceGroup: &managedClusterSpec.NodeResourceGroupName,
+			EnableRBAC:        to.BoolPtr(true),
```

Comment:
have you verified that the default is true?

Reply:
Yes, I created an AKS cluster without `EnableRBAC` set, and verified that the created cluster has kubernetes RBAC enabled.

However we cannot guarantee that the default behavior won't change, but I think it probably makes sense to always enable it explicitly? If you think we should allow user to choose and default to enable, I can make the change :)

Reply:
I'm good with enabling explicitly, and I don't foresee the default changing anytime soon. If a use case comes up to disable it, we can make it user-configurable then. For now, let's keep the change as-is.